templates/default/config/siteConfig.js

```diff
@@ -9,7 +9,10 @@ const defaultConfig = {
   // Google analytics key e.g. G-XXXX
   analytics: '',
   // content source directory for markdown files
-  content: process.env.FLOWERSHOW_CONTENT || 'content',
```

Comment:
if we remove content entirely won't things break?

Reply:
@rufuspollock, it's not really removed, this line has been changed. See lines below.